extensions/cornerstone-dynamic-volume/src/getHangingProtocolModule.ts

```diff
@@ -1,9 +1,259 @@
-function volumeDisplaySetMatcher() {
+const DEFAULT_COLORMAP = 'hsv'; // hsv, jet, rainbow_2, 2hot, Black-Body Radiation
```

Comment:
I guess your prettier is not working, can you make sure it is setup correctly? this files should be formatted